Engine/Component/ComponentCamera.cpp

```diff
@@ -27,8 +27,10 @@ ComponentCamera::ComponentCamera(GameObject * owner) : Component(owner, Componen
 ComponentCamera::~ComponentCamera()
 {
 	glDeleteTextures(1, &last_recorded_frame_texture);
-	glDeleteFramebuffers(1, &fbo);
 	glDeleteRenderbuffers(1, &rbo);
+	glDeleteFramebuffers(1, &fbo);
+	glDeleteFramebuffers(1, &msfbo);
+	glDeleteFramebuffers(1, &msfb_color);
```

Comment:
`msfb_color` is not a framebuffer, but a texture. So `msfb_color` should be deleted using `glDeleteTextures`